### DISCUSSION TITLE
Add to docs: with the default config in the README, where is the configuration saved?

### DISCUSSION THREAD

**Discussion by DeflateAwning** [open-webui/docs#140] [Q&A] [answered]
Please add this info to the docs (README): with the default config (default docker run command) in the README, where is the configuration saved?

**Comment by dusdoom** [comment]
Some of my friends went through this problem earlier when trying to store their stuff in another drive.
I think we should probably change the volume to a bind mount instead of a module or at least give a reference for the user to understand the difference between both of them.

**Comment by DeflateAwning** [comment]
I'm disappointed this was converted to a discussion. This is a documentation deficiency (an issue).

**Reply by justinh-rahb** [reply, marked as answer]
Config/user data is stored in a volume, per the docs:

![Screenshot 2024-06-28 at 2 58 25 PM](https://github.com/open-webui/docs/assets/52832301/3ee0a952-29ee-4736-9982-3814ac7c7b3b)

**Reply by marcosebsilva** [reply]
i added that section in the docs because of this issue and forgot to mention it
hope that it does the work

**Reply by DeflateAwning** [reply]
That was added to the docs since this discussion was opened. Thanks!
